job-descriptions/software-engineer.md

```diff
@@ -8,15 +8,17 @@ Sourcegraph's mission is to enable every software developer to create products u
 
 Our values:
 
-- **Inclusivity**. To build a product that serves the needs of all developers, we need to have a diverse team. Sourcegraph is an equal opportunity workplace; we embrace diversity and welcome people from all backgrounds and communities.
+- **Inclusivity**. To build a product that serves the needs of all developers, we are building a diverse team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we embrace diversity and welcome people from all backgrounds and communities.

```

Comment:
```suggestion
- **Inclusivity**. To build a product that serves the needs of all developers, we are building a diverse team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
```

What do you think about additionally removing the "we embrace diversity" clause? My thinking is that it seems to stutter the use of "diversity" in the previous sentence.

Reply:
Agree! Like your suggested change here.